apps/emqx/test/emqx_bpapi_static_checks.erl

```diff
@@ -82,7 +85,7 @@ run() ->
                     check_compat(Dumps)
             end;
         false ->
-            ?CRITICAL("Backplane API violations found on the current branch."),
```

Comment:
In the old macros there was logic provided to allow for defaults when we didn't pass in any args as a way to make things easier. Is there a reason that decision changed to force the passing of an empty list when there are no args?

Reply:
there is only one such macro in this module, so I took the easy way.